x/concentrated-liquidity/fees_test.go

```diff
@@ -62,37 +62,36 @@ func (s *KeeperTestSuite) TestInitializeFeeAccumulatorPosition() {
 	s.Require().NoError(err)
 
 	type initFeeAccumTest struct {
-		name           string
-		liquidityDelta sdk.Dec
-		positionId     positionIdentifiers
+		name       string
+		positionId positionIdentifiers
 
 		expectedPass bool
 	}
 	tests := []initFeeAccumTest{
 		{
-			name:           "new zero liqudidity position",
-			liquidityDelta: sdk.ZeroDec(),
-			// Gets overwritten by the next test case
+			name:         "first position with zero liquidity",
 			positionId:   defaultPositionId,
 			expectedPass: true,
 		},
 		{
-			name: "new non-zero liqudidity position",
-			// Does not get overwritten by the next test case
-			// because contains non-zero liquidity.
-			liquidityDelta: defaultLiquidityDelta,
-			positionId:     defaultPositionId,
-			expectedPass:   true,
+			name:         "second position with non-zero liquidity",
+			positionId:   withLowerTick(defaultPositionId, DefaultLowerTick+1),
+			expectedPass: true,
```

Comment:
its not clear to me how increasing the lowerTick by one makes the position have non-zero liquidity

Reply:
The names were applicable to the old test cases. Updated now